src/passes/Asyncify.cpp

```diff
@@ -663,6 +691,11 @@ class ModuleAnalyzer {
           if (matched) {
             info.addedFromList = true;
           }
+          if (verbose) {
+            std::cout << "[asyncify] " << func->name
+                      << "'s state is set based on the only-list to " << matched
+                      << '\n';
+          }
```

Comment:
Is looks possible for some functions to be logged as added as can-change-state functions but if the only-list is present, they are eventually ignored silently, i.e., without any other log messages that they are ignored, right? Would it be better to check the only list when logging for other causes..?

Reply:
I think this log message here would be shown, though, so it's not completely silent? This message would say they are set to `false`, which overrides the previous messages.

Reply:
Ah you're right. Sorry I thought this is only printed when the value is true.